members.ts

```diff
@@ -341,4 +341,16 @@ export const members: Member[] = [
     githubUsername: "senohirona",
     websiteUrl: "",
   }, 
+  {
+    id: "sakama",
+    name: "sakama",
+    role: "SRE",
+    bio: "homo sapiens",
+    avatarSrc: "/avatars/sakama.jpeg",
+    sources: [""],
```

Comment:
こちら空配列にしていただけますと！ 🙇‍♂️ 🙇‍♂️ 

```suggestion
    sources: [],
```

Reply:
承知しました！